03.asynchronous/callback.js

```diff
@@ -0,0 +1,70 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  (err) => {
+    if (err === null) {
+      db.run("INSERT INTO books(title) VALUES('fjord')", function (err) {
+        if (err === null) {
+          console.log(this.lastID);
+        } else {
+          console.error(err);
+        }
+
+        db.get("SELECT * from books WHERE id = ?", this.lastID, (err, row) => {
```

Comment:
SQL のキーワード部分は大文字で書いてください。

```suggestion
        db.get("SELECT * FROM books WHERE id = ?", this.lastID, (err, row) => {
```